TweetBook/Domain/Post.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace TweetBook.Domain
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TweetBook.Domain
 {
     public class Post
     {
+
+        [Key]
+
```

Comment:
The [Key] is called the key attribute. We put attributes between "[ ]" symbols. This attribute here tells us that the Id field from this class is the unique identifier for the Post db table(primary key). One attribute is usually strictly linked to its parameter, so please keep them close and erase the line between.